packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -161,14 +161,14 @@ Generated by [AVA](https://avajs.dev).
 
 ## makes usdc advance
 
-> Under "published", the "fastUsdc.status" node is delegated to the statuses of fast USDC transfers identified by their tx hashes.
+> Under "published", the "fastUsdc.txns" node is delegated to the Ethereum transactions upon which Fast USDC is acting.
```

Comment:
```suggestion
> Under "published", the "fastUsdc.txns" node is delegated to the Ethereum transactions upon which Fast USDC is acting.
```

I suppose `txns` is an abbreviation for `transactions`. Why not spell it out?

Reply:
1) concision
2) the evidence uses `txn`
3) to make clear this is an Ethereum-identified transaction

Reply:
ok, so `txns` isn't an abbreviation of transactions; it's a plural of the evidence property name.

Re `txn` distinguing between ethereum and others such as cosmos-sdk... I seem to have a gap in my education. If you have a moment to point me at what I missed, I'd appreciate it.

Reply:
> Re txn distinguing between ethereum and others such as cosmos-sdk

I retract #3. I didn't think it through carefully.

`txs` is the common Ethereum abbreviation for "transactions" and it's also used in Cosmos.

So I don't have a good reason to use `txns`. I do prefer `txs` to `transactions` but could go either way if you have a principle for the latter.

Reply:
using conventional dictionary words for identifiers that have scopes larger than a few lines pays off, IME.

"txn" is a sufficiently established term-of-art that I find it acceptable.